utils/datetime/LocalDate.js

```diff
@@ -178,6 +183,13 @@ export class LocalDate {
         }
     }
 
+    /**
+     * @returns {LocalDate} - the same date if already a weekday, or the next weekday.
+     */
+    currentOrNextWeekday() {
+        return this.isWeekday ? this : this.previousWeekday().nextWeekday();
```

Comment:
trying to understand why you need the previousWeekday() bit.

Reply:
Yes, sorry about that - was not trying to gaslight you!

That was the code Ii had in my app w/o `isWeekday` - to just always run the date back and forwards to ensure it landed on a weekday.  Didn't mean to leave that here - thanks for cleaning up.